src/apiProxy/apiProxyRoutes.js

```diff
@@ -41,7 +41,7 @@ const getApiProxyRoutes = (path, env, apiProxyFn$) => {
 	const apiPostRoute = {
 		path,
 		handler,
-		method: 'POST',
+		method: ['POST', 'DELETE'],
```

Comment:
to keep things simple, I think it's better to keep `DELETE` as part of the `GET` route - it's fine to send the payload as part of the querystring and then you don't have to worry about `delete` as an additional special case in `fetchQueries`